src/pages/tutor-profile/TutorProfile.jsx

```diff
@@ -7,23 +7,35 @@ import VideoPresentation from '~/containers/tutor-profile/video-presentation/Vid
 import CommentsWithRatingBlock from '~/containers/tutor-profile/comments-with-rating-block/CommentsWithRaitngBlock'
 
 import { responseMock } from '~/pages/tutor-profile/constants'
+import { useAppSelector } from '~/hooks/use-redux'
+import useUserInfo from '~/hooks/use-user-info'
+import Loader from '~/components/loader/Loader'
 
 const TutorProfile = () => {
   const { user } = responseMock
```

Comment:
remove this mock with real data

Reply:
Fixed.

Reply:
I can't completely delete this mock. The user object doesn't contain all the necessary data, so I need to leave this mock for now.